one_fm/hooks.py

```diff
@@ -164,7 +164,7 @@
 		"on_submit": "one_fm.utils.leave_appillication_on_submit",
 		"validate": [
 			"one_fm.utils.validate_hajj_leave",
-			"one_fm.one_fm.hr_utils.validate_leave_proof_document_requirement",
```

Comment:
It should be retain as it is. This is not purely related to indemnity, It is related to the Leave Type and Leave Application

Reply:
I'll add it back to hr_utils.py